src/decompiler-ast.inl

```diff
@@ -204,41 +206,102 @@ struct AST {
     NewNode(NodeType::Expr, e.type(), &e, arity.nargs);
   }
 
+  void FlushVars(int nargs, int num_vars, int dip, int num_vals) {
```

Comment:
Odd that dip was a default value before, but is now always included and not the last argument.

Reply:
To me, default values should preferrably not be used, unless the function in question is called many times and the vast majority of callers should want the default. That is the case for `InsertNode` and not `FlushVars`.